joplin/templates/wagtailadmin/pages/coa_header.html

```diff
@@ -0,0 +1,21 @@
+{% load wagtailadmin_tags staticfiles i18n %}
```

Comment:
adding this back cause the header is still potentially used on some pages and will look weird without them